Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.42100&attempt_id=e6a9ecc2-9f3d-4512-a798-888cf0298c44)

Attempted: copyq

<details><summary>Partial log (click to expand)</summary><p>

```
-- Found X11: /nix/store/56x9x8b9a2i3nbwd1i7q9bxx079ghpmf-libX11-1.6.5/lib/libX11.so
-- Building for X11 window system.
CMake Error at src/CMakeLists.txt:89 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".


-- Configuring incomplete, errors occurred!
See also "/build/source/build/CMakeFiles/CMakeOutput.log".
builder for '/nix/store/rvn8b6vipxc6dd6whwjw403vrj1nc08z-CopyQ-3.4.0.drv' failed with exit code 1
error: build of '/nix/store/rvn8b6vipxc6dd6whwjw403vrj1nc08z-CopyQ-3.4.0.drv' failed
```
</p></details>

--- a/pkgs/applications/misc/copyq/default.nix
+++ b/pkgs/applications/misc/copyq/default.nix
@@ -1,22 +1,23 @@
-{ stdenv, fetchFromGitHub, cmake, qtbase, qtscript, qtwebkit, libXfixes, libXtst, git
+{ stdenv, fetchFromGitHub, cmake, qtbase, qtscript, qtwebkit, libXfixes, libXtst
+, qtx11extras, git
 , webkitSupport ? true
 }:
 
 stdenv.mkDerivation rec {
   name = "CopyQ-${version}";
-  version = "3.3.1";
+  version = "3.5.0";
 
   src  = fetchFromGitHub {
     owner = "hluk";
     repo = "CopyQ";
     rev = "v${version}";
-    sha256 = "1jjb979dwdnkjca95yxzapbjpd6hr97hxz7sn614whvdv4vvpnyc";
+    sha256 = "0hzdv6rhjpq9yrfafnkc6d8m5pzw9qr520vsjf2gn1819gdybmr0";
   };
 
   nativeBuildInputs = [ cmake ];
 
   buildInputs = [
-    git qtbase qtscript libXfixes libXtst
+    git qtbase qtscript libXfixes libXtst qtx11extras
   ] ++ stdenv.lib.optional webkitSupport qtwebkit;
 
   meta = with stdenv.lib; {

